DIY Reminders with Azure and Twilio/proposal.md

```diff
@@ -0,0 +1,25 @@
+### :pushpin: Step 1
+**TITLE:**
+{DIY Alarms}
+
+**TOPIC**
+{Azure with Twilio integration}
+
+**DESCRIPTION**
+{This covers everything from the creation of the app in the Azure portal to the final testing after Twilio integration. Twilio usage can be covered if it's too vague without further explanation. There will be a bit of coding in javascript to handle the text and calls for Twilio and to catch errors, but those are simpler and all the code will be provided for the readers. The text and call integration parts will be kept separate so the reader can choose one or both of them to implement. Further explanations can be included for setting up Azure, and new images will be used to make sure they are relevant.}
+
+### :pushpin: Step 2
+:family: **TARGET AUDIENCE**
+{no required background, some javascript knowledge would be nice but not needed, anyone who wants to make their own call/text alarm}
+
+### :pushpin: Step 3
+> Outline learning/teaching structure
+
+**Beginning**
+{I can be quite forgetful when I'm tired or having fun, and it's not uncommon for me to miss a notification. I love camping, hiking, and biking, but I'm allergic to plants and need to take meds for it else it'll be a bad day for me. We all forget things from time to time, but you can code your own little bit more forceful call alarm.}
+
+**Middle**
+{Before we start, we need to have an active Microsoft Azure account which we can sign up for free for 12 months. We'll cover all the required setup including how to create a function app in Azure. We'll also make sure the user is good to go and has everything they need for this project. There will be a step by step walkthrough of the code firstly in Azure, then the Twilio integration. This will also have separate sections to set up text and call notification through Twilio.}
```

Comment:
Would recommend including a section that tells the user what technology they will be using before starting